src/utils/slpReader.ts

```diff
@@ -226,14 +234,46 @@ export function iterateEvents(
       return readPosition;
     }
 
+    const advanceAmount = buffer.length;
+
     readRef(ref, buffer, 0, buffer.length, readPosition);
+    if (commandByte === Command.SPLIT_MESSAGE) {
+      // Here we have a split message, we will collect data from them until the last
+      // message of the list is received
+      const view = new DataView(buffer.buffer);
+      const size = readUint16(view, 0x201) ?? 512;
+      const isLastMessage = readBool(view, 0x204);
+      const internalCommand = readUint8(view, 0x203) ?? 0;
```

Comment:
Over here too, what when would it be undefined to need a default value of `0`?

Reply:
Same as before, the `readUint8` function can return null, this ensures that the value of the variable ends up being a number.

Reply:
Probably the more correct thing to do is to check for null and throw an error maybe? Or skip processing? But I think by setting the command to zero it will skip processing anyway. But yeah idk, I agree it could be slightly confusing.